packages/cli/README.md

```diff
@@ -334,9 +347,175 @@ ARGUMENTS
 
 OPTIONS
   -c, --ceramic=ceramic  Ceramic API URL
-  -k, --key=key    DID Key
+  -k, --key=key          DID Key
+```
+
+### `glaze pin:add STREAMID`
+
+pin a stream
+
+```
+USAGE
+  $ glaze pin:add STREAMID
+
+ARGUMENTS
+  STREAMID  ID of stream to be pinned
+
+OPTIONS
+  -c, --ceramic=ceramic  Ceramic API URL
+  -k, --key=key          DID Key
+```
+
+### `glaze pin:ls [STREAMID]`
+
+list pinned streams
+
+```
+USAGE
+  $ glaze pin:ls [STREAMID]
+
+ARGUMENTS
+  STREAMID  optional stream ID filter
+
+OPTIONS
+  -c, --ceramic=ceramic  Ceramic API URL
+  -k, --key=key          DID Key
+```
+
+### `glaze pin:rm STREAMID`
+
+unpin a stream
+
+```
+USAGE
+  $ glaze pin:rm STREAMID
+
+ARGUMENTS
+  STREAMID  ID of stream to be unpinned
+
+OPTIONS
+  -c, --ceramic=ceramic  Ceramic API URL
+  -k, --key=key          DID Key
+```
+
+### `glaze stream:commits STREAMID`
+
+list commits contained within a stream
+
+```
+USAGE
+  $ glaze stream:commits STREAMID
+
+ARGUMENTS
+  STREAMID  ID of the stream
+
+OPTIONS
+  -c, --ceramic=ceramic  Ceramic API URL
+  -k, --key=key          DID Key
+```
+
+### `glaze stream:state STREAMID`
+
+get the state of a Stream
+
+```
+USAGE
+  $ glaze stream:state STREAMID
+
+ARGUMENTS
+  STREAMID  ID of the Stream
+
+OPTIONS
+  -c, --ceramic=ceramic  Ceramic API URL
+  -k, --key=key          DID Key
+```
+
+### `glaze tile:create CONTENT`
+
+create a new Tile stream
+
+```
+USAGE
+  $ glaze tile:create CONTENT
+
+ARGUMENTS
+  CONTENT  stream contents (JSON encoded as string)
+
+OPTIONS
+  -c, --ceramic=ceramic    Ceramic API URL
+  -g, --only-genesis       only generate genesis commit
+  -k, --key=key            DID Key
+  -m, --metadata=metadata  stream metadata
+```
+
+### `glaze tile:deterministic METADATA`
+
+load a deterministic Tile stream
+
+```
+USAGE
+  $ glaze tile:deterministic METADATA
+
+ARGUMENTS
+  METADATA  stream metadata
+
+OPTIONS
+  -c, --ceramic=ceramic  Ceramic API URL
+  -k, --key=key          DID Key
 ```
 
+### `glaze tile:show STREAMID`
+
+show the contents of a Tile stream
+
+```
+USAGE
+  $ glaze tile:show STREAMID
+
+ARGUMENTS
+  STREAMID  ID of the stream
+
+OPTIONS
+  -c, --ceramic=ceramic  Ceramic API URL
+  -k, --key=key          DID Key
+```
+
+### `glaze tile:update STREAMID CONTENT`
+
+Update a stream
+
+```
+USAGE
+  $ glaze tile:update STREAMID CONTENT
+
+ARGUMENTS
+  STREAMID  ID of the stream
+  CONTENT   new contents for the stream
```

Comment:
how would you use the update command to just change the metadata but not the content?  Maybe content should be an option flag both here and in the create command?  That's how it works in the Ceramic cli today

Reply:
Yes good point, cc @Sterahi

Reply:
Great idea. will implement that now.